cmd/kubebuilder/initproject/project.go

```diff
@@ -44,6 +43,12 @@ type projectOptions struct {
 }
 
 func (o *projectOptions) RunInit() {
+	checkGoVersion()
```

Comment:
Does it make sense to combine this file with `init.go`?

Reply:
Good point, just did that.